identity-account/src/tests/lazy.rs

```diff
@@ -27,102 +28,126 @@ async fn test_lazy_updates() -> Result<()> {
       // ===========================================================================
       // Create, update and publish an identity
       // ===========================================================================
-      let account: Account = Account::builder().autopublish(false).build().await?;
 
       let network = if test_run % 2 == 0 {
         Network::Devnet
       } else {
         Network::Mainnet
       };
 
-      let identity: IdentityState = account
-        .create_identity(IdentityCreate::new().network(network.name()).unwrap())
-        .await?;
-
-      let did: &IotaDID = identity.try_did()?;
+      let config = AccountConfig::default().autopublish(false);
+      let account_config = AccountSetup::new(Arc::new(MemStore::new())).config(config);
```

Comment:
Nit: could we use the `testmode` flag in the lazy `Account` updates test to avoid having to communicate with the Tangle or is real publishing/resolving still required? Optional, fine to leave as-is too.

Reply:
Thanks for making me double-check. I got rid of the tangle dependency.